api/client/build.go

```diff
@@ -209,6 +210,15 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		}
 	}
 
+	var shmSize int64 = 67108864 // initial SHM size is 64MB
```

Comment:
IMO, either way it'd be helpful if the help text listed what the default is, so I'm personally probably more in favor of defaulting the option to the default value since that makes it obvious that this is a flag that doesn't add new constraints (like `--cpu-shares` or `-m`), but adjusts an existing constraint.